### DISCUSSION TITLE
Capture people entering and leaving door

### DISCUSSION THREAD

**Discussion by riskendemism** [blakeblackshear/frigate#3091] [Ask A Question]
Hello there,

I have a question about zones.

My setup is a camera pointed at a door, right and left of the door the camera captures about 3 meters of space. My goal is to only detect persons who are entering this door or are leaving the door. I wish no detecting from persons who only pass by. 

Is there any chance this can be accomplished with the help of zoning?

Thank you in advance and greetings from Vienna!
Cheers
Chris

**Comment by NickM-27** [comment]
It should be possible with a zone. An object being inside a zone is based on the bottom center of the bounding box, so you'll want to create a zone where their feet would be if coming through the door but not walking by.

**Reply by riskendemism** [reply]
The camerapicture looks like the picture attached. 

Everyone would pass through the "feet" zone. Or maybe I dont quite understand what you are proposing...
![grey-lh-security-steel-door](https://user-images.githubusercontent.com/64709469/162960027-dbf9162e-5185-4243-8e82-616c4a655d93.jpg)

**Reply by NickM-27** [reply]
I would try something like this, and it can be slightly bigger (towards bottom of image) or smaller depending on how much you don't want it to trigger on people passing by.

![162960027-dbf9162e-5185-4243-8e82-616c4a655d93](https://user-images.githubusercontent.com/14866235/162960612-44be9c41-e7af-4559-8e74-7eeef764638e.jpeg)

**Reply by riskendemism** [reply]
But still, when someone just walks by, parts of the body are still in your defined zone.

**Reply by NickM-27** [reply]
> But still, when someone just walks by, parts of the body are still in your defined zone. 

Zones work based on the bottom center of the bounding box being in the zone, not overlapping

**Reply by NickM-27** [reply]
Consider this drawn out example
![162960612-44be9c41-e7af-4559-8e74-7eeef764638e](https://user-images.githubusercontent.com/14866235/163424642-32fc405c-6167-4c02-8179-74f3edcffb36.jpeg)

**Comment by renarena** [comment]
> being inside a zone is based on the **bottom center of the bounding box**
